components/sidebar.vue

```diff
@@ -214,6 +214,18 @@
           this.$cookies.get('userPicture') || require('../assets/avatar.svg')
         );
       },
+      isNl() {
+        return this.$i18n.locale === 'nl';
+      },
+      shouldShowNovelties() {
+        if (this.isNl) {
+          return true;
+        }
+        if (this.$config.onlyShowNlNovelties.toString() === 'true') {
```

Comment:
Code changes look great, just going to wait for the outcome of this conversation about the need for this config parameter before approving: https://github.com/cultuurnet/appconfig/pull/11#issuecomment-710448722

Reply:
done